docs/man/tutorials.rst

```diff
@@ -138,26 +139,30 @@ This tutorial is provided in the WEC-Sim code release in the ``$WECSIM/tutorials
 
 Device Geometry
 ^^^^^^^^^^^^^^^^^^^^^^^
-The OSWEC was selected because its design is fundamentally different from the RM3. This is critical because WECs span an extensive design space, and it is important to model devices in WEC-Sim that operate under different principles.  The OSWEC is fixed to the ground and has a flap that is connected through a hinge to the base that restricts the flap in order to pitch about the hinge. The full-scale dimensions of the OSWEC and the mass properties are shown in the figure and table below.
+The OSWEC was selected because its design is fundamentally different from the RM3. This is critical because WECs span an extensive design space, and it is important to model devices in WEC-Sim that operate under different principles.  The OSWEC is fixed to the ground and has a flap that is connected through a hinge to the base that restricts the flap in order to pitch about the hinge.
+The full-scale dimensions and mass properties of the OSWEC are shown below.
 
 .. figure:: /_static/images/OSWEC_Geom.png
    :width: 300pt
    :align: center
 
-+-----------------------------+
-|Flap Full Scale Properties   |
-+======+=========+============+
-|      |         |Pitch Moment|
-+CG (m)+Mass (kg)+of Inertia  +
-|      |         |(kg-m^2)    |
-+------+---------+------------+
-|  0   |         |            |
-+------+         +            +
-|  0   |127,000  |1,850,000   |
-+------+         +            +
-| -3.9 |         |            |
-+------+---------+------------+
++-------------------+----------------------------------+
+| Mass |nbsp| (kg)  | Pitch Moment of Inertia (kg m^2) |
++===================+==================================+
+| 727.01            | 1,850,000                        |
++-------------------+----------------------------------+
+
++-----------+------------------------+
+| Direction | Center of Gravity* (m) |
++===========+========================+
+|    x      |  0                     |
++-----------+------------------------+
+|    y      |  0                     |
++-----------+------------------------+
+|    z      | -3.9                   |
++-----------+------------------------+
```

Comment:
I suggest using the same format as the RM3 tables above for consistency. Otherwise, I agree that dividing the tables into components that depend on x,y,z (I,cg) and those that don't (mass) is more clear when reading.